charts/apps/jupyter-web-app/templates/ConfigMap/jupyter-web-app-parameters-42k97gcbmb-kubeflow-ConfigMap.yaml

```diff
@@ -11,5 +11,5 @@ metadata:
   labels:
     app: jupyter-web-app
     kustomize.component: jupyter-web-app
-  name: jupyter-web-app-parameters-dhcbt5dtdf
+  name: jupyter-web-app-parameters-42k97gcbmb
```

Comment:
Is this used anywhere?

Reply:
https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#generatoroptions

Reply:
this suffix is auto-generated by the helm chart generator